### ISSUE TITLE
save scores per day per user name

### ISSUE THREAD

**Issue by umar-muneer** [umar-muneer/go-careaxiom-services#6]
this can help us do the following

1. support modification.
2. allow only a single review per day.

**Comment by WaleedAshraf** [comment]
does it mean to save separate score for each user?

**Comment by umar-muneer** [comment]
Yep.. Seems like a safer approach then saving review count in the sheet. I
am afraid we will run into race conditions in the existing approach
